app/imports/api/actionItems/schema.js

```diff
@@ -1,6 +1,10 @@
 import SimpleSchema from 'simpl-schema';
 
 const ActionItemsSchema = new SimpleSchema({
+    projectId: {
+        type: String,
+        defaultValue: '',
```

Comment:
Always must have `projectId`, remove `defaultValue`. It must be provided by user/method.